OpenRA.Mods.Common/Traits/AutoCarryable.cs

```diff
@@ -25,6 +25,10 @@ public class AutoCarryableInfo : CarryableInfo
 	public class AutoCarryable : Carryable, ICallForTransport
 	{
 		readonly AutoCarryableInfo info;
+		bool autoCommandReserved = false;
```

Comment:
```suggestion
		bool checkCarryRange = false;
```

If it was renamed it would make the code easier to read and understand. The comment could also be removed

Reply:
`checkCarryRange` is not match line 127 of check the carry range again.

Reply:
change to `autoReserved` as other place changed.